internal/monitors/haproxy/metadata.yaml

```diff
@@ -59,6 +59,10 @@ monitors:
       unique_proxy_id:
         description: 'A unique proxy id'
     metrics:
+      haproxy_status:
+        default: false
+        description: '. Values reported for listeners, frontends, backends and servers.'
```

Comment:
Leading `.`.

Reply:
Also this description is quite vague.  What kind of values is this supposed to be?  Is it some kind of enum?

Reply:
Ok, better now but you need to remake the docs.

Reply:
Remade the docs